resolwe_bio/processes/chip_seq/macs14.yml

```diff
@@ -72,14 +55,37 @@
       label: Negative peaks (XLS)
       type: basic:file
       required: false
+    - name: species
+      label: Species
+      type: basic:string
+    - name: build
+      label: Build
+      type: basic:string
   run:
     runtime: polyglot
     language: bash
     program: |
       NAME=`basename {{ treatment.bam.file }} .bam`
+      
+      SPECIES={{ treatment.species }}
+      {% set dict = {
+        'Homo sapiens': '2.7e9',
+        'Mus musculus': '1.87e9',
+        'Dictyostelium discoideum': '3.4e7',
+        'Drosophila melanogaster': '1.2e8',
+        'Caenorhabditis elegans': '9e7',
+        'Rattus norvegicus': '2e9',
+        }
+      %}
+      GSIZE={{ dict[treatment.species] }}
+      SPECIES_LIST="{% for species in dict.keys() %}{{species}}, {% endfor %}"
+      SPECIES_LIST=${SPECIES_LIST%??} 
+      if [ -z $GSIZE ]; then
+        re-error "${SPECIES} is not a valid hybrid species. Accepted species are: ${SPECIES_LIST}."
+      fi
 
       CONTROL='{% if control %}--control {{ control.bam.file }} {% endif %}'
-      macs --treatment {{ treatment.bam.file }} ${CONTROL} --format BAM --gsize {{ gsize }} --name $NAME --pvalue {{ pvalue }} --wig --single-profile --space=50
+      macs --treatment {{ treatment.bam.file }} ${CONTROL} --format BAM --gsize ${GSIZE} --name $NAME --pvalue {{ pvalue }} --wig --single-profile --space=50
```

Comment:
Revrite into more readable form, like [here](https://github.com/genialis/resolwe-bio/blob/master/resolwe_bio/processes/alignment/star.yml#L208).

Reply:
Also note that code from #400 is duplicated here (regarding `upload-bam` and `upload-bam-indexed` process)